<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="KARNA" failures="1" tests="11" name="Test" time="129.439" errors="0" timestamp="08 Mar 2024 06:19:36 GMT">
  <testcase classname="com.cps.qa.testCases.CustomerTransactionPage_Test" name="Verify_Deposit_With_ValideAmmount" time="2.898"/>
  <testcase classname="com.cps.qa.testCases.CustomerLogin_Test" name="customerlogin" time="2.521"/>
  <testcase classname="com.cps.qa.testCases.CustomersPage_Test" name="DeleteCustomerEntry" time="1.26"/>
  <testcase classname="com.cps.qa.testCases.AddCustomerPage_Test" name="AddCustomer_NotExist" time="2.198"/>
  <testcase classname="com.cps.qa.testCases.OpenAccountPage_Test" name="OpenAccount" time="12.941"/>
  <testcase classname="com.cps.qa.testCases.AccountPage_Test" name="Verify_Deposit_With_ValideAmmount" time="4.306"/>
  <testcase classname="com.cps.qa.testCases.AddCustomerPage_Test" name="AddCustomer_Exist" time="4.056"/>
  <testcase classname="com.cps.qa.testCases.AccountPage_Test" name="Verify_Deposit_With_InValideAmmount" time="2.516">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.cps.qa.pages.AccountPage.InValidDeposit(AccountPage.java:179)
	at com.cps.qa.testCases.AccountPage_Test.Verify_Deposit_With_InValideAmmount(AccountPage_Test.java:49)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Verify_Deposit_With_InValideAmmount -->
  <testcase classname="com.cps.qa.testCases.AccountPage_Test" name="Verify_Withdraw_With_ValideAmmount" time="15.649"/>
  <testcase classname="com.cps.qa.testCases.AccountPage_Test" name="Verify_Withdraw_With_InValideAmmount" time="12.95"/>
  <testcase classname="com.cps.qa.testCases.AccountPage_Test" name="Verify_Withdraw_With_InSufficient" time="15.873"/>
</testsuite> <!-- Test -->
